src/pyrad_proc/pyrad/io/io_aux.py

```diff
@@ -417,7 +427,22 @@ def get_file_list(datadescriptor, starttime, endtime, cfg, scan=None):
             dayfilelist = glob.glob(datapath+dayinfo+'*'+datatype+'.nc')
             for filename in dayfilelist:
                 t_filelist.append(filename)
-
+        elif datagroup == 'MXPOL':
+            if scan is None:
+                warn('Unknown scan name')
+                return []
```

Comment:
Do you have something to check for coding convention.. I did try to follow as much as possible the same style as your scripts... except for maybe some variable names...